packages/repository-tests/src/crud/relations/fixtures/models/address.model.ts

```diff
@@ -1,20 +1,26 @@
 // Copyright IBM Corp. 2019. All Rights Reserved.
-// Node module: @loopback/repository
+// Node module: @loopback/repository-tests
 // This file is licensed under the MIT License.
 // License text available at https://opensource.org/licenses/MIT
 
-import {belongsTo, Entity, model, property} from '../../..';
+import {belongsTo, Entity, model, property} from '@loopback/repository';
 import {Customer, CustomerWithRelations} from './customer.model';
+import {mixedIdType} from '../../helpers';
 
 @model()
 export class Address extends Entity {
+  @property({
+    type: 'number',
```

Comment:
I think this needs to be `features.idType`?

Reply:
We didn't change the model files to `create{modelName}model` functions.